pkgs/top-level/aliases.nix

```diff
@@ -563,7 +563,6 @@ mapAliases ({
   ruby_2_3 = throw "ruby_2_3 was deprecated on 2019-09-06: use a newer version of ruby";
   ruby_2_4_3 = throw "ruby_2_4_3 was deprecated on 2018-02-13: use a newer version of ruby";
   ruby_2_4 = throw "ruby_2_4 was deprecated in 2019-12: use a newer version of ruby";
-  ruby_2_5_0 = throw "ruby_2_5_0 was deprecated on 2018-02-13: use a newer version of ruby";
```

Comment:
Please revert that change. Otherwise people run into eval errors.

Reply:
Oh, of course, I wasn't thinking.

Should I add similar deprecation error lines to this file for `ruby_2_5` and `rubyPackages_2-5`?

Reply:
Yes please.